torch/csrc/jit/passes/onnx/remove_inplace_ops_for_onnx.cpp

```diff
@@ -335,6 +336,7 @@ static void PrepareForRemoveMutations(MutationRemover& mr, Block* b) {
         TORCH_INTERNAL_ASSERT(nullptr != newNode);
         node->replaceInput(index, newNode->output());
         input->replaceAllUsesAfterNodeWith(node, newNode->output());
+        goto restart;
```

Comment:
This is also a good point. Please don't use `goto`, can we use `break` here?

Reply:
I see you change the code to remove `goto`, and this `needsRestart` makes sense to me.